gradle/scripts/tscfg.gradle

```diff
@@ -29,35 +29,6 @@ task genConfigClass {
   }
 }
 
-task genExperimentsConfigClass {
-  doLast {
-    def tscfgJarFile = project.file('build/tscfg-' + tscfgVersion + '.jar')
-    if (!tscfgJarFile.exists() || !tscfgJarFile.isFile()) {
-      download {
-        src 'https://github.com/carueda/tscfg/releases/download/v' + tscfgVersion + '/tscfg-' + tscfgVersion + '.jar'
-        dest buildDir
-      }
-    }
-    javaexec {
-      main = "-jar"
-      args = [
-        "build/tscfg-${tscfgVersion}.jar",
-        "--spec",
-        "src/main/resources/config/experiments-config-template.conf",
-        "--scala",
-        "--durations",
-        "--pn",
-        "edu.ie3.simona.config",
-        "--cn",
-        "ExperimentsConfig",
-        "--dd",
-        "src/main/scala/edu/ie3/simona/config/"
-      ]
-    }
-  }
-}
-
-
```

Comment:
Why? Does not relate to #130 nor #148

Reply:
I just recognized in the [discussion with sebastian](https://github.com/ie3-institute/simona/pull/131#discussion_r809149780) that this is code, that was meant to be introduced with #66, but due to some error already is in code base right now.